altair/vegalite/v5/compiler.py

```diff
@@ -0,0 +1,26 @@
+from ..compiler import VegaLiteCompilerRegistry
+
+vegalite_compilers = VegaLiteCompilerRegistry()
+
+
+def vl_convert_compiler(vegalite_spec) -> dict:
```

Comment:
```suggestion
def vl_convert_compiler(vegalite_spec: dict) -> dict:
```

Reply:
done in f043b485107ba02643755df873554978596b3482